dlt-database/entity/0001-init_db/TransactionRecipe.ts

```diff
@@ -63,7 +55,7 @@ export class TransactionRecipe {
   @Column({ type: 'tinyint' })
   type: number
 
-  @CreateDateColumn({ name: 'created_at', type: 'datetime' })
+  @Column({ name: 'created_at', type: 'datetime' })
```

Comment:
If the `createdAt` column will only contain the timestamp of the write access than the annotation `@CreateDateColumn` will be prefered.
But keep in mind:
the `@CreateDateColumn` and/or `@UpdateDateColumn` will not treat the ms-dimension properly